src/main/java/org/apache/maven/plugins/enforcer/EnforceBytecodeVersion.java

```diff
@@ -272,6 +272,8 @@ private String isBadArtifact( Artifact a )
             jarFile = new JarFile( f );
             getLog().debug( f.getName() + " => " + f.getPath() );
             byte[] magicAndClassFileVersion = new byte[8];
+            StringBuilder artifactInfo = new StringBuilder();
+            artifactInfo.append( "(" ).append( a.getGroupId() ).append( ":" ).append( a.getArtifactId() ).append( ") " );

```

Comment:
```suggestion
            String artifactInfo =  "("  + a.getGroupId() + ":"  + a.getArtifactId() + ") ";
```

Seems simpler as you do not extend the `StringBuilder` lateron?